h2o-algos/src/main/java/hex/schemas/GAMV3.java

```diff
@@ -232,10 +232,10 @@
     //public BSType bs;
 
     @API(help="Save keys of model matrix", level = Level.secondary, direction = Direction.INPUT)
-    public boolean save_gam_cols; // if true will save keys storing GAM columns
+    public boolean keep_gam_cols; // if true will save keys storing GAM columns
     
     @API(help="String arrays storing frame keys of knots.  One for each gam column specified in gam_x", 
```

Comment:
Last one to change here.
"String arrays storing frame keys of knots.  One for each GAM column specified in gam_columns",

Reply:
Good catch Angela.  Fixed